catalog/spatial/wfs/spatial-wfs-converter/src/main/java/org/codice/ddf/spatial/ogc/wfs/catalog/converter/impl/GenericFeatureConverter.java

```diff
@@ -245,4 +243,8 @@ public void setSourceId(final String sourceId) {
         this.sourceId = sourceId;
     }
 
+    private boolean isAttributeSet(Attribute attr) {
```

Comment:
How about doing this instead?
```
private void ifAttributeSet(Attribute attribute, Consumer<Attribute> consumer) {
    if (attr != null && attr.getValue() != null) {
        consumer.accept(attribute);
    }
}
```

Then all your checks above can be written as:
```
ifAttributeSet(
    mc.getAttribute(Core.CREATED),
    mc::setAttribute(new AttributeImpl(Core.CREATED, genericDate)));
```